pkg/k8s/api/v1alpha1/edgedevice_types.go

```diff
@@ -47,8 +47,9 @@ type OPCUASetting struct {
 
 // SocketSetting defines Socket specific settings when connecting to an EdgeDevice
 type SocketSetting struct {
-	Encoding    *string `json:"encoding,omitempty"`
-	NetworkType *string `json:"networkType,omitempty"`
+	Encoding     *string `json:"encoding,omitempty"`
+	BufferLength *int    `json:"bufferLength,omitempty"`
```

Comment:
Can you please look into CRD default values? we should be able to set default values here directly 
https://book.kubebuilder.io/reference/markers/crd-validation.html